recipe/meta.yaml

```diff
@@ -27,22 +31,20 @@ test:
     - redis
```

Comment:
Can we extend `test/imports`? See https://github.com/redis/redis-py/blob/e6cd4fdf3b159d7f118f154e28e884069da89d7e/setup.py#L15-L21